.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -23,3 +23,7 @@ Is your feature request related to a problem? Please describe the problem here.
 ### Additional context
 
 Add any other context or screenshots about the feature request here. For example, did this FR come from https://discuss.streamlit.io or another site? Link the original source here!
+
+### Voting for feature requests
```

Comment:
nit: maybe it's better to just have a divider here so that it is a bit clearer that it is not a section that people need to actively fill out.

```suggestion
---

```

Reply:
that makes sense!